airflow/providers/google/cloud/hooks/cloud_storage_transfer_service.py

```diff
@@ -97,8 +97,15 @@ class GcpTransferOperationStatus:
 TRANSFER_JOBS = 'transferJobs'
 TRANSFER_OPERATIONS = 'transferOperations'
 TRANSFER_OPTIONS = 'transfer_options'
-TRANSFER_SPEC = 'transferSpec'
+TRANSFER_SPEC: str = 'transferSpec'
```

Comment:
Why did you only specify the type here? Why is the type not added for the other constants?

Reply:
Seems it was automatic. I'll remove this type